### PR TITLE
Enable windows-7 32-bit jobs for Metricbeat

### PR DESCRIPTION
**Pull request by kvch**
## What does this PR do?

This PR enabled the job `x-pack/metricbeat-windows-7-32-windows-7-32-bit` which had to be turned off temporarily when introducing Golang 1.15.

When x-pack/metricbeat jobs run on a 32-bit Windows, coverage and DWARF are disabled, so do not run out of memory.

## Why is it important?

Tests for x-pack/metricbeat on Windows 7 32bit version were disabled. Now they are available again.

## Checklist

- [x] My code follows the style guidelines of this project
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have made corresponding change to the default configuration files~~
- [x] I have added tests that prove my fix is effective or that my feature works
~~- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

### CODE REVIEW

**Comment by kvch** [comment]
jenkins run tests

**Review by kuisathaverat — APPROVED** [review]

File: dev-tools/mage/gotest.go [hunk 1]
```diff
@@ -68,6 +69,10 @@ func makeGoTestArgs(name string) GoTestArgs {
 		JUnitReportFile: fileName + ".xml",
 		Tags:            testTagsFromEnv(),
 	}
+	if runtime.GOOS == "windows" {
+		fmt.Println("hallo")
```
Comment (cachedout) [inline dev-tools/mage/gotest.go:73]: Remove debugging? :)
```suggestion
```

File: Jenkinsfile [hunk 2]
```diff
@@ -348,7 +348,7 @@ def withBeatsEnv(Map args = [:], Closure body) {
     "PATH=${path}",
     "PYTHON_ENV=${pythonEnv}",
     "RACE_DETECTOR=true",
-    "TEST_COVERAGE=true",
+    //"TEST_COVERAGE=true", TODO
```
Comment (v1v) [inline Jenkinsfile:351]: What's the reason?

Comment (kvch) [reply Jenkinsfile:351]: It was just a test to see if the issue still persist if the coverpkg flag is removed.

**Review by cachedout — APPROVED** [review]

**Comment by kvch** [comment]
jenkins run tests

File: dev-tools/mage/integtest.go [hunk 3]
```diff
@@ -247,6 +248,10 @@ func initRunner(tester IntegrationTester, dir string, passInEnv map[string]strin
 		env["MAGEFILE_VERBOSE"] = "1"
 	}
 
+	if runtime.GOOS == "windows" {
+		env["TEST_COVERAGE"] = "false"
+	}
```
Comment (urso) [inline dev-tools/mage/integtest.go:253]: Why disable coverage information for integration tests? I though system tests are the only problem. We might also want to test the architecture. AFIAK only 32bit windows is having problems, right?

Comment (kvch) [reply dev-tools/mage/integtest.go:253]: Every test seems to be impacted, not only system tests.

Comment (urso) [reply dev-tools/mage/integtest.go:253]: > Every test seems to be impacted, not only system tests.

I see :/

Still we might want to disable TEST_COVERAGE only if the tests are run on a 32bit system.

**Comment by kvch** [comment]
jenkins run tests

File: dev-tools/mage/integtest.go (hunk 3 above)
Comment (kvch) [reply dev-tools/mage/integtest.go:253]: OK.

Comment (kvch) [reply dev-tools/mage/integtest.go:252]: Also, I will only disable it for Metricbeat....

**Review by urso — APPROVED** [review]

**Review by v1v — APPROVED** [review]

**Pull request merged**